client/src/remote/Plugins.js

```diff
@@ -92,8 +95,8 @@ export default class Plugins {
     };
 
     // Carbon React exports for the client plugins
-    global.carbon = CarbonReactExports;
-    global.carbonicons = CarbonIconsReactExports;
+    vendor.carbon = CarbonReactExports;
```

Comment:
We _could_ change this, because it was just recently introduced, and not publicly exposed.